ts/src/bitget.ts

```diff
@@ -3967,6 +3968,53 @@ export default class bitget extends Exchange {
         return this.filterByArray (result, 'symbol', symbols, false);
     }
 
+    async fetchPositionsBySymbol (symbol, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        if (!market['linear'] || !market['swap']) {
```

Comment:
@ttodua let's not do this, we should implement for the available markets upon adding the method